### ISSUE TITLE
Small project status bar UI tweak

### ISSUE THREAD

**Issue by ctwhome** [research-software-directory/RSD-as-a-service#933]
The current implementation shows a progress bar for each project as designed. After seeing the final result with the current data I think it looks rather cluttered. I have a proposal to keep the card a bit cleaner, while still keeping the message of the status: 
- Completed projects: show only starting and finish dates.
- In-progress projects: show dates and progress bar.
- Non-started projects: show dates and progress bar (gray bar)

As seen in the following example, the projects that are in progress are more prominent and they create a clearer distinction between completed and not started. As well as giving a less cluttered UI overall. 

![image](https://github.com/research-software-directory/RSD-as-a-service/assets/4195550/fce01e86-f6b9-4beb-9942-cd6a2a048ade)

**Comment by jmaassen** [comment]
Good idea. Do we always know when a project will start? If not, we may have an additional option for non-started projects: 

- "waiting to start" (when the start date is not known yet)
- "dates and grey progress bar" (when the start date is known and in the future)

**Comment by ctwhome** [comment]
> * "dates and grey progress bar" (when the start date is known and in the future)
Yes indeed, that is the state for Non started project (Last option)

"waiting to start" is also a good case, no progress bar in that case, only the label. 

I updated the design to include these options!

**Comment by ctwhome** [comment]
Reopening the issue with updated cases where a project is "ongoing" or doesn't have a finished date. 
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/4195550/6239d8dd-e4fd-4c3d-9781-841573743146)

**Comment by ewan-escience** [comment]
I think the only change we need is that a *finished* project with an unknown starting date should have a full progress bar.
